app/matchers/bulkrax/application_matcher.rb

```diff
@@ -101,6 +101,7 @@ def extract_model(src)
     end
 
     # Only add valid resource types
+    # @todo(bjustice) - remove hyrax reference NOT USED?
```

Comment:
Is this still a TODO?

Reply:
I don't think this method is used at all in Bulkrax.

Reply:
Hm, I'm not sure. I've asked [in Slack](https://assaydepot.slack.com/archives/G0313NK5NMA/p1679593992121479)